test/test-parse.cpp

```diff
@@ -3593,8 +3593,78 @@ TEST(test_parse, with_statement) {
 
 TEST(test_parse, break_statement) {
   {
-    spy_visitor v = parse_and_visit_statement(u8"break;"_sv);
+    spy_visitor v;
+    padded_string code(u8"break;"_sv);
+    parser p(&code, &v);
+    p.parse_and_visit_statement(v);
     EXPECT_THAT(v.visits, IsEmpty());
+    EXPECT_THAT(v.errors, ElementsAre(ERROR_TYPE_FIELD(
+                              error_invalid_break, break_statement,
+                              offsets_matcher(&code, 0, 5))));
+  }
+
+  {
+    spy_visitor v;
+    padded_string code(u8"for (;;) { } break;"_sv);
+    parser p(&code, &v);
+    p.parse_and_visit_statement(v);
+    p.parse_and_visit_statement(v);
+    EXPECT_THAT(v.errors, ElementsAre(ERROR_TYPE_FIELD(
+                              error_invalid_break, break_statement,
+                              offsets_matcher(&code, 13, 18))));
```

Comment:
Nit: New convention for offsets_matcher:
```suggestion
                              offsets_matcher(&code, strlen(u8"for (;;) { } "), u8"break"))));
```